packages/api/bin/serve.js

```diff
@@ -250,7 +255,37 @@ async function serveDistributionApi(stackName = 'localrun', done) {
   return distributionApp.listen(port, done);
 }
 
+/**
+ * Removes all additional data from tables and repopulates with original data.
+ *
+ * @param {string} user - defaults to local user, testUser
+ * @param {string} stackName - defaults to local stack, localrun
+ */
+async function resetTables(user = 'testUser', stackName = 'localrun') {
```

Comment:
Maybe `testUser` should be stored in a variable, like `testUsername` to keep it in sync between this file and testAuth.js?